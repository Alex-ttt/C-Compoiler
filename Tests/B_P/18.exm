 Symbols Table:
   main  void function main(
  )
    {
      Statements:
      a
      [
        1
      ]
      Symbols Table:
              a array[10] of int
    }
