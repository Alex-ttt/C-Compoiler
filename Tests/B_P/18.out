Expected open bracket '('	 line 1, col 1

